Library/Homebrew/cmd/update.sh

```diff
@@ -219,8 +219,16 @@ pull() {
 
   if [[ -z "$HOMEBREW_MERGE" ]]
   then
+    if [[ -n "$HOMEBREW_VERBOSE" ]]
+    then
+      echo "Rebasing $DIR onto origin/$UPSTREAM_BRANCH ..."
+    fi
     git rebase "${QUIET_ARGS[@]}" "origin/$UPSTREAM_BRANCH"
   else
+    if [[ -n "$HOMEBREW_VERBOSE" ]]
+    then
+      echo "Merging origin/$UPSTREAM_BRANCH into $DIR..."
```

Comment:
Sure, in normal text I always use “…” (it's even directly accessible from my keyboard layout). It looks kinda weird in output on the command line though (squeezed into a single column) and can be problematic in source code, where we tend to stick to ASCII-only.